public/css/drag-drop-style.css

```
@@ -0,0 +1,523 @@
+  color: white;
+  padding: 10px 50px;
+  border-radius: 38px;
+}
+
+.answerButton {
+  margin-top: 16px;
+  padding: 16px 10px;
+  color: rgb(60, 67, 162);
+  min-height: 32px;
+
+  cursor: pointer;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)